tests/cleaning/quality_filter_test.py

```diff
@@ -0,0 +1,218 @@
+        når det er over 28,5 meter i sekundet.
+
+        Andre dele af landet har nu også mærket de første vindstød af stormstyrke. 
+
+        Odense Lufthavn har haft 24,5 meter i sekundet, mens Grønlandshavnen i Aalborg har ramt 24,7
+        meter i sekundet. Det er mest centrale sted i landet, hvor der indtil videre er målet stormstyrke.
+        """
+
+    @pytest.fixture(scope="class")
+    def bullets_text(self):
+        return """
+        [summary]
+
+        - test 1
+        - test 1
+            - test 2
+            - test 3
+                - test 1
+        """
+
+    @pytest.fixture(scope="class")
+    def ellipsis_text(self):
+        return """
+        [summary]
+
+        * test 1
+            * test 2
+            * test 3
+        * test 1
+            * test 2
+                * test 3
+        """
+
+    @pytest.fixture(scope="class")
+    def all_texts(self, tweet_texts, long_text, bullets_text, ellipsis_text):
+        return tweet_texts + [long_text] + [bullets_text] + [ellipsis_text]
+
+    @pytest.fixture(scope="class")
+    def stop_words(self):
+        return set(
+            [
+                "er",
+                "jeg",
+                "det",
+                "du",
+                "ikke",
+                "at",
+                "en",
+                "og",
+                "har",
+                "vi",
+                "til",
+                "på",
+                "hvad",
+                "mig",
+                "med",
+                "de",
+                "for",
+                "den",
+                "så",
+                "der",
+                "dig",
+                "han",
+                "kan",
+                "af",
+            ]
+        )
+
+    @pytest.fixture(scope="class")
+    def quality_filter(self):
+        return QualityFilter()
+
+    def test_good_stop_words(self, quality_filter, stop_words):
+        assert (
+            quality_filter.stop_word(
+                quality_filter.nlp("jeg er glad"), n=2, stop_words=stop_words
+            )
+            is True
+        )
+
+    def test_bad_stop_words(self, quality_filter, stop_words):
+        assert (
+            quality_filter.stop_word(
+                quality_filter.nlp("56789okd23456789098765sds"),
+                n=1,
+                stop_words=stop_words,
+            )
+            is False
+        )
+
+    def test_good_bullets(self, quality_filter):
+        assert (
+            quality_filter.line_bullets_or_ellipsis(
+                quality_filter.nlp("56789okd23456789098765sds"),
+                max_p_bullets=0.5,
+                max_p_ellipsis=1,
+            )
+            is True
+        )
+
+    def test_bad_bullets_on_ellipsis(self, quality_filter, ellipsis_text):
+        assert (
+            quality_filter.line_bullets_or_ellipsis(
+                quality_filter.nlp(ellipsis_text), max_p_bullets=0.5, max_p_ellipsis=1
+            )
+            is False
+        )
```

Comment:
Again this is False because of the bullet point part, not the ellipsis part (i.e. this and the next test just test two different types of bullets)